Assets/HoloToolkit-Examples/SpatialMapping/SpatialProcessingSurfaceMagnetism.md

```diff
@@ -12,17 +12,11 @@ https://gfycat.com/PitifulRareKodiakbear
 ## How to use 
 You need to assign same Unity layer to the Spatial mapping’s plane and Solver Surface Magnetism’s Magnetic Surface.
 
-1. Add a new layer for the planes created by Spatial Processing. Mixed Reality Toolkit's **Apply Mixed Reality Projcet Settings** automatically addes a new layer **31:Spatial Mapping**.
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism1.png" width="550">
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism2.png" width="300">
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism3.png" width="550">
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism4.png" width="300">
+1. Add a new layer for the planes created by Spatial Processing. Mixed Reality Toolkit's **Apply Mixed Reality Projcet Settings** automatically addes a new layer **31:Spatial Mapping**. <img src="/External/ReadMeImages/MRTK_SurfaceMagnetism1.png" width="550"> <img src="/External/ReadMeImages/MRTK_SurfaceMagnetism2.png" width="300"> <img src="/External/ReadMeImages/MRTK_SurfaceMagnetism3.png" width="550"> <img src="/External/ReadMeImages/MRTK_SurfaceMagnetism4.png" width="300">
 
-2. In SpatialMapping prefab, make sure to set the Physics Layer to 31.
-<img src="/External/ReadMeImages/MRTK_SurfaceMagnetism5.png" width="550">
+2. In SpatialMapping prefab, make sure to set the Physics Layer to 31. <br/><img src="/External/ReadMeImages/MRTK_SurfaceMagnetism5.png" width="550">
```

Comment:
This is the default within the prefab. I would reword step 2 to say "make sure the Physics Layer is set to 31"